cluster-autoscaler/hack/update-vendor-osx.sh

```diff
@@ -0,0 +1,219 @@
+#!/usr/local/bin/bash
```

Comment:
I added this file in a quick manner to be able to run it on OSX and Linux. We could remove it in the future.